### DISCUSSION TITLE
How to Stop or Uninstall AdGuard from Console in OPNsense

### DISCUSSION THREAD

**Discussion by AseKarlsson** [AdguardTeam/AdGuardHome#6895] [Q&A]
Hi,

Made a stupid mistake and installed ADGuard Home on my OPNsense box on port 80.
This means that I cannot access my OPNsense anymore that used the port, only ADGuatd Home answers.

I can still access the console/shell och my OPNsense box, so how do I either stop the ADGuard Home service or uninstall it?

Thanks,
Ase

**Comment by AseKarlsson** [comment]
I could restart OPNsense and access the OPNsense GUI before ADGuard Home had started and solved it that way.

**Comment by ghost** [comment]
Stop AdGuard Home

- Log in to your OPNsense admin interface.
- Navigate to the Services section.
- Find AdGuard Home in the list of services.
- Click on the Stop button next to AdGuard Home to halt its operation.

To change the AdGuard Home http interface port:

- Open the AdGuardHome.yaml configuration file in your preferred text editor.
- Locate the http section of the configuration file.
- You will find an address setting within this section. Change the port number in this setting to your desired port. For example, if you want to set the port to 8080, you would change it to 0.0.0.0:8080 to listen on all network interfaces, or 127.0.0.1:8080 to listen only on the local loopback interface1.
- Save the changes to the AdGuardHome.yaml file.
